src/main/resources/config/reflectionconfig-java.json

```diff
@@ -14,7 +14,9 @@
     "name":"java.lang.Long"
   },
   {
-    "name":"java.lang.Object"
+    "name":"java.lang.Object",
+    "allDeclaredFields":true,
```

Comment:
I think this has a big impact on footprint?

Reply:
I took it from the agent. Actually there are no fields in Object, so probably it is not even required?